lib/Installation/DiskActivation/ZFCPConfigurationPage.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to the Configured ZFCP dialog dialog
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::DiskActivation::ZFCPConfigurationPage;
```

Comment:
I understand the inheritance of the press_next method. I'm still struggling to get the concept of new. If I remove new from the child class then new from the parent class (in this case NavigationBase) will be used when invoking Child::new(). This new calls a method init() but does that mean NavigationBase::init() or Chil::init(): 
I'm a bit  confused because LicenseAgreementPage.pm still has  a new method, so there should be a reason for this.

Reply:
init is overwritten and use the child one, it is call I think initialization pattern, help to not mess up with the constructor much and have that inheritiance for initialization. The reason there are new() method exactly the same in parent and child when there is this inheritance from Navigation is that we just forget to delete them.